exampleSite/config.toml

```diff
@@ -68,14 +68,33 @@ enableEmoji = true
   textFontFamily = "sans-serif"
   headingFontFamily = "sans-serif"
 
+  # Specify use of utterances or giscus
+  [params.comments]
+    use_utterances = false
+    use_giscus = true
```

Comment:
Do you think it'd be possible to have a single top-level `comments = "giscus"` or `comments = "utterances"` (i.e. `{{ params.comments }}` in the template) that would control this option? I don't think people would enable more than one commenting system and it also leaves room for future commenting platforms to be added. (On that note, it might make sense to choose Disqus via this param as well.)

Reply:
yes! Just as I was reading your first comment, that was my first though immediately! 

I also think it makes sense to control disqus through this option too. Making it, as you say, easier to add more options in the future.